readme.md

```diff
@@ -151,8 +151,13 @@ Tokens
 
 ### Creating a token
 
-A token can be used anywhere on Stripe where you would normally pass a card. Once it's created, it can be used on a
-customer or a charge, but only used once.
+A token can be used anywhere on Stripe where you would normally pass a card.
+Once it's created, it can be used on a customer or a charge, but only used once.
+
+In production, you'll want to create tokens with either
+[stripe.js](https://stripe.com/docs/tutorials/forms) or
+[Checkout](https://stripe.com/docs/tutorials/checkout), but it can be useful
+to create tokens with Stripe.net for testing.
```

Comment:
there are situations where a PCI compliant server might want a token on a checkout page, then might want to use that token on another page. If users decide to not use the two libraries you mentioned above because they are compliant, now we might be recommending they store credit cards in their system. Either session state, or worse, database.